### DISCUSSION TITLE
How to can catch the json parse error in the body

### DISCUSSION THREAD

**Discussion by hitokiri** [mockoon/mockoon#490] [Help] [answered]
Hi,
I have the next problem; I need to catch the JSON malformed on the Body request; the APP only shows a popup in the right corner. but how can enlaced this event for to create a ruler when that happening

**Comment by 255kb** [comment, marked as answer]
Hi, there is currently no way to specifically react to invalid entering JSON. Feel free to open a new feature request so we can track this.
